config/settings.py

```diff
@@ -547,8 +547,8 @@
 
 # Debug Toolbar
 # -----------------------------------------------------------------------------
-DEBUG_TOOLBAR_PATCH_SETTINGS = False
```

Comment:
This setting has been deprecated for some time: https://django-debug-toolbar.readthedocs.io/en/latest/changes.html#id31